src/components/pages/style.scss

```diff
@@ -30,29 +30,25 @@
 	&__page-header {
 		height: 35px;
 		padding: 10px;
-		color: #82878c;
+		color: #555d66;
```

Comment:
It'd be good to start a variables.scss (or borrow the one from Gutenberg).

Reply:
Let's just copy this:
https://github.com/WordPress/gutenberg/blob/master/assets/stylesheets/_colors.scss

Reply:
@draganescu is doing this in #102.